### ISSUE TITLE
Unexpected low performance for recipes/TIMIT/Alignment

### ISSUE THREAD

**Issue by weiwei-ww** [speechbrain/speechbrain#1175]
I was running the DNN-HMM model under recipes/TIMIT/Alignment and got a normal accuracy (>80%). However, once I changed the phoneme set (e.g. from 60 phonemes to 39 phonemes), the performance drops significantly (with ~30% accuracy), no matter how I tuned the hyperparameters (e.g. learning rate, etc.). Is there anything that I have missed?

**Comment by mravanelli** [comment]
It's a long time I don't work on this task. However, I remember that it was quite hard making things converging. @lorenlugosch or @elenaras might share more insights here.

**Comment by lorenlugosch** [comment]
Yes, our HMM aligner was super unstable, and we haven't looked at it in a while.

If you need a good forced aligner to get frame-level phoneme labels, I'd recommend the Montreal Forced Aligner. There is also CTC-Segmentation (speechbrain/alignment/ctc_segmentation.py) in SpeechBrain, but I haven't tried it.

**Comment by weiwei-ww** [comment]
@mravanelli @lorenlugosch Thanks for your comments here!
